client/attest_test.go

```diff
@@ -0,0 +1,110 @@
+package client
```

Comment:
For this package we should add two additional types of tests:

1. A test that actually calls `client.Attest` to make sure the `ValidateCertChain` option works correctly.
2. A test that calls `client.Attest` with ValidateCertChain = true`, but where the key does not have a certificate.
  a. We should decide whether to error or do-nothing in this case.